src/frontend/src/CustomNodes/GenericNode/index.tsx

```diff
@@ -343,6 +337,7 @@ export default function GenericNode({
                   showNode={showNode}
                   validationStatus={validationStatus}
                   isOutdated={isOutdated}
+                  beta={data.node?.beta!}
```

Comment:
hey can we not use the ! on this, and use someDefault value to avoid crashes with components